src/main/java/io/reactivex/flowables/ConnectableFlowable.java

```diff
@@ -34,8 +34,12 @@
  * @param <T>
  *          the type of items emitted by the {@code ConnectableObservable}
  */
-public abstract class ConnectableFlowable<T> extends Flowable<T> {
-    
+public abstract class ConnectableFlowable<T> extends FlowableWithUpstream<T, T> {
+
+    protected ConnectableFlowable(Publisher<T> source) {
```

Comment:
Ok I'm going to send a new PR with that change for existing operators, and then circle back for these connectable folk.